src/test/java/uk/gov/register/store/postgres/PostgresDataAccessLayerTest.java

```diff
@@ -40,12 +41,13 @@
     private ObjectMapper objectMapper = new ObjectMapper();
 
     @Rule
-    public WipeDatabaseRule wipeDatabaseRule = new WipeDatabaseRule(TestRegister.address);
+    public RegisterRule register = new RegisterRule();
```

Comment:
this was a similar issue to that in https://github.com/openregister/openregister-java/pull/564#discussion_r244588862 whereby the test attempted to manipulate database state which not exist depending on what order tests are run.